### PR TITLE
updated elemental analysis ref data

### PR DESCRIPTION
**Pull request by AnthonyLim23**
**Description of work.**
We are updating the elemental analysis peak data file. The user has provided it and we are uploading it to be the new default. The old one had errors. 

**To test:**
Open Elemental analysis
Right click Fe - a pop up will appear
Scroll down in the new window to the Gamma section
You will see the same transition multiple times (with additional spaces preceding it)

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by NickDraper — COMMENTED** [review]
Confirmed that this only changes the XML file, and no others.

Just building locally to test functionality

**Comment by NickDraper** [comment]
I do think we should mention this in the release notes, as the same analysis with different versions of Mantid would give different answers. @AnthonyLim23

**Comment by AnthonyLim23** [comment]
I will once this has gone in https://github.com/mantidproject/mantid/pull/28294

File: scripts/Muon/GUI/ElementalAnalysis/peak_data.json [hunk 1]
```diff
@@ -3,2370 +3,2463 @@
...
-      "g54Mn25 (1)": 54.5,
-      "g54Mn25 (2)": 156.2,
-      "g54Mn25 (3)": 212,
-      "g54Mn25 (4)": 251.6,
-      "g54Mn25 (5)": 376.9,
-      "g55Mn25": 126
+      "g54Mn (100)": 54.5,
+      "g54Mn(100)": 156.2,
+      " g54Mn (100)": 212,
+      "  g54Mn(100)": 251.6,
```
Comment (NickDraper) [inline scripts/Muon/GUI/ElementalAnalysis/peak_data.json]: Are the whitespace differences here intentional?  They are carried through into the GUI and look messy.

File: scripts/Muon/GUI/ElementalAnalysis/peak_data.json [hunk 2]
```diff
@@ -3,2370 +3,2463 @@
...
+      "L(3d->2p)  ": 1895.5,
+      "L(3d->2p)   ": 1920.7,
+      "L(3d->2p)    ": 1977,
+      "L(3d->2p)     ": 2048.1,
+      "M(5f->3d)": 1050.5,
+      "M(5f->3d) ": 1074.3,
+      "M(6f->3d)": 1230.1,
+      "M(6f->3d) ": 1254.8,
+      "N(6g->4f)": 618.1,
+      "N(6g->4f) ": 623.2,
```
Comment (NickDraper) [inline scripts/Muon/GUI/ElementalAnalysis/peak_data.json]: more trailing whitespace

File: scripts/Muon/GUI/ElementalAnalysis/peak_data.json [hunk 3]
```diff
@@ -3,2370 +3,2463 @@
...
+    "Secondary": {
+      "L(3d->2p)": 2360.4,
+      "M(5f->3d)": 1301.1,
+      "M(5f->3d) ": 1336.3,
+      "M(6f->3d)": 1520.4,
+      "M(6f->3d) ": 1562.2,
+      "N(6g->4f)": 630.6,
+      "N(6g->4f) ": 637.5,
+      "N(7g->4f)": 764.7,
+      "N(7g->4f) ": 771.9,
```
Comment (NickDraper) [inline scripts/Muon/GUI/ElementalAnalysis/peak_data.json]: again

File: scripts/Muon/GUI/ElementalAnalysis/peak_data.json [hunk 4]
```diff
@@ -3,2370 +3,2463 @@
...
+      "L(3d->2p)    ": 1781.7,
+      "L(3d->2p)     ": 1803.0,
+      "M(5f->3d)": 905.5,
+      "M(5f->3d) ": 911.2,
+      "M(5f->3d)  ": 923.1,
+      "M(5f->3d)   ": 929.9,
+      "N(6g->4f)": 441.6,
+      "N(6g->4f)  ": 444.8,
+      "N(7g->4f)": 534.4,
+      "N(7g->4f) ": 538.2,
```
Comment (NickDraper) [inline scripts/Muon/GUI/ElementalAnalysis/peak_data.json]: Is this intentional?

File: scripts/Muon/GUI/ElementalAnalysis/peak_data.json [hunk 5]
```diff
@@ -3,2370 +3,2463 @@
...
-      "M(6->3)": 566.7,
-      "M(7->3)": 573.5,
-      "6->4": 276.3,
-      "7->4": 334.5
+      "K(3p->1s)": 4819.2,
+      "L(3d->2p)": 1091.6,
+      "L(4d->2p)": 1491.4,
+      "L(4d->2p) ": 1541.4,
+      "M(5f->3d)": 566.7,
+      "M(5f->3d) ": 573.5,
```
Comment (NickDraper) [inline scripts/Muon/GUI/ElementalAnalysis/peak_data.json]: it happens quite a lot

File: scripts/Muon/GUI/ElementalAnalysis/peak_data.json [hunk 6]
```diff
@@ -3,2370 +3,2463 @@
...
+      "L(4d->2p)": 1272.7,
+      "L(4d->2p) ": 1278.4,
+      "L(5d->2p)": 1316.7,
+      "L(6d->2p)": 1424.9,
+      "L(7d->2p)": 1431.4,
+      "L(8d->2p)": 1470.2,
+      "M(5f->3d)": 484,
+      "M(5f->3d) ": 488.9,
+      "M(6f->3d)": 567,
+      "M(6f->3d) ": 571.9,
```
Comment (NickDraper) [inline scripts/Muon/GUI/ElementalAnalysis/peak_data.json]: I'm going to stop highlighting them now - there are most certainly others ...

**Review by NickDraper — DISMISSED** [review]

**Comment by AnthonyLim23** [comment]
Yes the extra spaces are intentional, we want the same label but with different values. So the easiest way to tell them apart was to add extra spaces

**Comment by NickDraper** [comment]
So we agreed that the extra spaces can stay, but we should be consistent about whether the spaces are leading or trailing the label

**Comment by martyngigg** [comment]
I'm not suggesting we hold this up but for the future is it worth considering using nested lists rather than whitespace to distinguish this? It would also avoid repetition and typing errors with the key, e.g
```json
      "L(3d->2p)": [1858.5, 1882.2, 1895.5, 1920.7,1977, 2048.1]
```
would associate all values with `"L(3d->2p)"` and then whatever reads it could decide how to format them.

**Comment by AnthonyLim23** [comment]
In the short term they want them all available and putting them into a list will mean significant changes in how the data is read. The plan is to allow for different isotopes (thats why the same transition appears multiple times). We will probably do this in October. 

I am currently waiting on more information as the new file has no H data. I want to check this is correct.

**Comment by NickDraper** [comment]
removing myself as tester so other can pick it up while I'm not around.

File: scripts/Muon/GUI/ElementalAnalysis/peak_data.json [hunk 7]
```diff
@@ -3,2370 +3,2463 @@
...
-      "M(11->3)": 875.6,
-      "6->4": 331.5,
-      "7->4": 401.5,
-      "6->5": 116,
-      "7->5": 186.1,
-      "8->5": 231
+      "K(2p1/2->1s1/2)": 4085.3,
+      "K (2p1/2->1s1/2)": 4097.4,
+      "K(2p3/2->1s1/2)": 4157.9,
+      "K (2p3/2->1s1/2)": 4170.7,
```
Comment (Harrietbrown) [inline scripts/Muon/GUI/ElementalAnalysis/peak_data.json]:
```suggestion
      "K(2p3/2->1s1/2) ": 4170.7,
```

File: scripts/Muon/GUI/ElementalAnalysis/peak_data.json [hunk 8]
```diff
@@ -3,2370 +3,2463 @@
...
-      "M(9->3)": 804.9,
-      "M(10->3)": 865.3,
-      "M(11->3)": 875.6,
-      "6->4": 331.5,
-      "7->4": 401.5,
-      "6->5": 116,
-      "7->5": 186.1,
-      "8->5": 231
+      "K(2p1/2->1s1/2)": 4085.3,
+      "K (2p1/2->1s1/2)": 4097.4,
```
Comment (Harrietbrown) [inline scripts/Muon/GUI/ElementalAnalysis/peak_data.json]:
```suggestion
      "K(2p1/2->1s1/2) ": 4097.4,
```

File: scripts/Muon/GUI/ElementalAnalysis/peak_data.json [hunk 9]
```diff
@@ -3,2370 +3,2463 @@
...
-      "L(10->2)": 364.5,
-      "L(11->2)": 373.3,
-      "L(12->2)": 376.5,
-      "M(5->3)": 115,
-      "M(6->3)": 134.9,
-      "M(7->3)": 146.9,
-      "6->4": 55.4,
-      "7->4": 67.4
+      "K (2p->1s)": 1089.3,
+      "K(2p->1s)": 1092.5,
```
Comment (Harrietbrown) [inline scripts/Muon/GUI/ElementalAnalysis/peak_data.json]:
```suggestion
      "K(2p->1s)": 1089.3,
      "K(2p->1s) ": 1092.5,
```

File: scripts/Muon/GUI/ElementalAnalysis/peak_data.json [hunk 10]
```diff
@@ -3,2370 +3,2463 @@
...
-      "M(7->3)": 215.6,
-      "M(8->3)": 234.4,
-      "M(9->3)": 239.8,
-      "5->4": 53.5,
-      "7->4": 99.8,
-      "8->4": 111.4
+      "K(2p1/2->1s1/2)": 1507.7,
+      "K(2p3/2->1s1/2)": 1514.2,
+      "L(3d->2p)": 330.9,
+      "L (3d->2p)": 336.6,
```
Comment (Harrietbrown) [inline scripts/Muon/GUI/ElementalAnalysis/peak_data.json]:
```suggestion
      "L(3d->2p) ": 336.6,
```

File: scripts/Muon/GUI/ElementalAnalysis/peak_data.json [hunk 11]
```diff
@@ -3,2370 +3,2463 @@
...
-      "g61Ni28": 67,
-      "g62Ni28": 1173,
-      "g63Ni28": 155,
-      "g63Ni28(1.7us)": 87.1,
-      "g65Ni28": 692.3
+      "g61Ni (100)": 67,
+      "g62Ni (100)": 1173,
+      "g62Ni (100) ": 875,
+      "g63Ni (100)": 155,
+      "g63Ni (100) ": 87.1,
```
Comment (Harrietbrown) [inline scripts/Muon/GUI/ElementalAnalysis/peak_data.json]:
```suggestion
      "g61Ni(100)": 67,
      "g62Ni(100)": 1173,
      "g62Ni(100) ": 875,
      "g63Ni(100)": 155,
      "g63Ni(100) ": 87.1,
```

File: scripts/Muon/GUI/ElementalAnalysis/peak_data.json [hunk 12]
```diff
@@ -3,2370 +3,2463 @@
...
+      "L (3d-> 2p)": 1703.9,      
+      "L (3d- > 2p)": 1743.7,      
+      "L(3d->2p )": 1761.8,
+      "L(3d->2p) ": 1767.2,
+      "M(5f->3d)": 882,
+      "M (5f->3d)": 898,
+      "M(6f->3d)": 1032,
+      "M (6f->3d)": 1051.1,
+      "N(6g->4f)": 428.4,
+      "N( 6g->4f)": 431.7,
```
Comment (Harrietbrown) [inline scripts/Muon/GUI/ElementalAnalysis/peak_data.json]:
```suggestion
      "K(2->1)": 4602.1,
      "K(2p->1s) ": 4610.8,
      "K(2p->1s) ": 4676.9,
      "K(2p->1s)  ": 4689.9,
      "K(2p->1s)   ": 4718.4,
      "K(2p->1s)    ": 4804.1,
      "K(2p->1s)     ": 4813.3,
      "L(3d->2p)": 1580.8,
      "L(3d->2p) ": 1604.9,
      "L(3d->2p)  ": 1622.6,
      "L(3d->2p)   ": 1659.9,
      "L(3d->2p)    ": 1674.7,
      "L(3d->2p)     ": 1703.9,      
      "L(3d->2p)      ": 1743.7,      
      "L(3d->2p)       ": 1761.8,
      "L(3d->2p)        ": 1767.2,
      "M(5f->3d)": 882,
      "M(5f->3d) ": 898,
      "M(6f->3d)": 1032,
      "M(6f->3d) ": 1051.1,
      "N(6g->4f)": 428.4,
      "N(6g->4f) ": 431.7,
```

File: scripts/Muon/GUI/ElementalAnalysis/peak_data.json [hunk 13]
```diff
@@ -3,2370 +3,2463 @@
...
-      "M(8->3)": 1096.4,
-      "M(9->3)": 1115.4,
-      "7->4": 455,
-      "8->4": 458.5,
-      "9->4": 551.2,
-      "10->4": 555,
-      "6->5": 160.2,
-      "7->5": 256.7
+      "K(2p->1s)": 4778.2,
+      "K (2p->1s)": 4957.7,
```
Comment (Harrietbrown) [inline scripts/Muon/GUI/ElementalAnalysis/peak_data.json]:
```suggestion
      "K(2p->1s) ": 4957.7,
```

File: scripts/Muon/GUI/ElementalAnalysis/peak_data.json [hunk 14]
```diff
@@ -3,2370 +3,2463 @@
...
-      "g54Mn25 (3)": 212,
-      "g54Mn25 (4)": 251.6,
-      "g54Mn25 (5)": 376.9,
-      "g55Mn25": 126
+      "g54Mn (100)": 54.5,
+      "g54Mn (100)": 156.2,
+      "g54Mn (100) ": 212,
+      "g54Mn(100)  ": 251.6,
+      "g54Mn25": 376.9,
+      "g55Mn25 (100)": 126
```
Comment (Harrietbrown) [inline scripts/Muon/GUI/ElementalAnalysis/peak_data.json]:
```suggestion
      "g54Mn(100)": 54.5,
      "g54Mn(100) ": 156.2,
      "g54Mn(100)  ": 212,
      "g54Mn(100)   ": 251.6,
      "g54Mn25": 376.9,
      "g55Mn25(100)": 126
```

File: scripts/Muon/GUI/ElementalAnalysis/peak_data.json [hunk 15]
```diff
@@ -3,2370 +3,2463 @@
...
-      "6->4": 226,
-      "7->4": 273.5,
-      "6->5": 127.4,
-      "7->6": 79.6
+      "K(2p1/2->1s1/2": 3226.4,
+      "K(2p3/2->1s1/2)": 3266.2,
+      "K(2p->1s)": 3231.2,
+      "K(2p->1s) ": 3237.3,
+      "K (2p->1s)": 3269.8,
+      "K( 2p->1s)": 3277.0,
```
Comment (Harrietbrown) [inline scripts/Muon/GUI/ElementalAnalysis/peak_data.json]:
```suggestion
      "K(2p->1s)": 3269.8,
      "K(2p->1s) ": 3277.0,
```

**Review by Harrietbrown — CHANGES_REQUESTED** [review]
There are still quite a few inconsistant formatting issues in this. I've marked as many as I can see.

**Comment by AnthonyLim23** [comment]
Thank you for spotting these. I will try to get them done later, but might be tomorrow.

File: scripts/Muon/GUI/ElementalAnalysis/peak_data.json [hunk 16]
```diff
@@ -3,2370 +3,2463 @@
...
+      "K(2p1/2->1s1/2)": 5776.8,
+      "K(2p1/2->1s1/2) ": 5784.5,
+      "K(2p3/2->1s1/2)": 5961.9,
+      "K(2p3/2->1s1/2) ": 5970.8,
+      "L(3d5/2->2p3/2)": 2501.1,
+      "L(3d3/2->2p1/2)": 2643.1,
+      "M(4f7/2->3d5/2)": 936.7,
+      "M(4f5/2->3d3/2)": 970.7,
+      "N(5g9/2->4f7/2)": 430.8,
+      "N(5g7/2->4f5/2)": 437.1,
```
Comment (Harrietbrown) [inline scripts/Muon/GUI/ElementalAnalysis/peak_data.json]: Are these out of order?

File: scripts/Muon/GUI/ElementalAnalysis/peak_data.json [hunk 17]
```diff
@@ -3,2370 +3,2463 @@
...
-    },
-    "Secondary": {
-      "K(4->1)": 5610.1,
-      "L(4->2)": 1425.3,
-      "L(5->2)": 1839.8,
-      "L(6->2)": 1912.3,
-      "L(7->2)": 2061.8,
-      "L(8->2)": 2131.4,
-      "M(6->3)": 703,
-
```
Comment (Harrietbrown) [inline scripts/Muon/GUI/ElementalAnalysis/peak_data.json]: is the `~` intential?

Comment (Harrietbrown) [inline scripts/Muon/GUI/ElementalAnalysis/peak_data.json]: are these out of order

Comment (Harrietbrown) [inline scripts/Muon/GUI/ElementalAnalysis/peak_data.json]: Are these out of order?

Comment (Harrietbrown) [inline scripts/Muon/GUI/ElementalAnalysis/peak_data.json]: Are these out of order

Comment (Harrietbrown) [inline scripts/Muon/GUI/ElementalAnalysis/peak_data.json]: Are these out of order?

Comment (Harrietbrown) [inline scripts/Muon/GUI/ElementalAnalysis/peak_data.json]: Are these out of order?

Comment (Harrietbrown) [inline scripts/Muon/GUI/ElementalAnalysis/peak_data.json]: Are these out of order?

**Review by Harrietbrown — CHANGES_REQUESTED** [review]
A few more whitespace issues. I also noticed that some cases with fractional energy levels are not in a consistant order. I don't know if they are important enough to fix but a consistent ordering would be good to have.

File: scripts/Muon/GUI/ElementalAnalysis/peak_data.json [hunk 18]
```diff
@@ -3,2370 +3,2463 @@
...
+      "K(2p3/2->1s1/2)": 2708.1,
+      "L(3d->2p)": 695,
+      "L(3d->2p) ": 717.3,
+      "M(4f->3d)": 243.1,
+      "M(4f->3d) ": 245.4,
+      "N(5g->4f)": 112.6
+    },
+    "Secondary": {
+      "K(2p->1s)": 2674,
+      "K(2p1/2->1s1/2)": 2683.5,
```
[the remark this answers is not in the document]
Comment (AnthonyLim23) [reply scripts/Muon/GUI/ElementalAnalysis/peak_data.json]: this is the order the scientist that uses the interface gave me, so I assume they are happy with this order.

File: scripts/Muon/GUI/ElementalAnalysis/peak_data.json [hunk 19]
```diff
@@ -3,2370 +3,2463 @@
...
+      "M(4f->3d)": 498.7,
+      "M(4f->3d) ": 508.8,
+      "N(5g->4f)": 230.4,
+      "O(6h->5g)": 124.4
+
+    },
+    "Secondary": {
+      "K(2p1/2->1s1/2)": 4241,
+      "K(2p3/2->1s1/2)": 4266.2,
+      "K(2p1/2->1s1/2) ": 4272.4,
```
[the remark this answers is not in the document]
Comment (AnthonyLim23) [reply scripts/Muon/GUI/ElementalAnalysis/peak_data.json]: Same as above.

File: scripts/Muon/GUI/ElementalAnalysis/peak_data.json (hunk 16 above)
Comment (AnthonyLim23) [reply scripts/Muon/GUI/ElementalAnalysis/peak_data.json]: same as above

File: scripts/Muon/GUI/ElementalAnalysis/peak_data.json (hunk 17 above)
Comment (AnthonyLim23) [reply scripts/Muon/GUI/ElementalAnalysis/peak_data.json]: I assume it is. It could signify an approximation

**Review by Harrietbrown — APPROVED** [review]
The list displays as expected. at some point in the future, as has been said, It would be good to replace these with lists in instances where the same change in energy level has different lines. Bur for this release this is good to go.
Good job

**Pull request merged**